### ISSUE TITLE
Inference with cvt_text for dependency parsing

### ISSUE THREAD

**Issue by alirezadir** [tensorflow/models#6292]
Hi @clarkkev and @lmthang , thank you so much for putting cvt code together. I have trained cvt_text for Dependency Parsing on OntoNotes5 data, and tried evaluating on my own dataset. However, I have been having a hard time figuring out how to decode predicted labels of depparse_scorer into the (head, dependency) tags. I appreciate your help.

**Comment by alirezadir** [comment]
If I am not wrong, the labels for each sentence are calculated as follows (except for root): 
len(label_mapping) * ( 1 + head ) + label_mapping [dep-tag]
I cannot figure out any way to solve for (head, dep-tag) from the above equation :(

**Comment by clarkkev** [comment]
Hi @alirezadir. We "flatten" the labels so a single softmax decides both the heads and the tags. You can go from the flattened labels to (head, dep-tag) as follows. If the label is 0, the head/tag is root. Otherwise, the head is (label // len(label_mapping)) - 1 and the dep-tag is label_mapping[label % len(label_mapping)]. The -1 in the conversion is because we prepend a start-of-sentence token to each sentence during preprocessing.
